mobilecoind/src/payments.rs

```diff
@@ -228,6 +242,11 @@ impl<T: UserTxConnection + 'static, FPR: FogPubkeyResolver + Send + Sync + 'stat
         Ok(tx_proposal)
     }
 
+    /// Create a TxProposal that attempts to merge multiple UTXOs into a single larger UTXO.
+    ///
+    /// # Arguments
+    /// * `monitor_id` - ???
```

Comment:
I believe this generates a transaction that joins a bunch of UTXOs belonging to monitor_id+subaddress_index